mozilla_version/gecko.py

```diff
@@ -177,6 +179,15 @@ def __attrs_post_init__(self):
                 ), (
                     self.patch_number is not None and self.is_aurora_or_devedition,
                     'Patch number cannot be defined on an aurora version',
+                ), (
+                    self.major_number > 54 and self.is_aurora_or_devedition,
+                    'Last aurora/devedition version was 54.0a2. Please use the DeveditionVersion '
+                    'class, past this version.',
+                ), (
+                    self.major_number not in self._KNOWN_ESR_MAJOR_NUMBERS and self.is_esr,
+                    '"{}" is not a valid ESR major number. Valid ones are: {}'.format(
+                        self.major_number, self._KNOWN_ESR_MAJOR_NUMBERS
```

Comment:
This will be very helpful if the job fails in the future with an unpredictable esr version cut. Nice! 👍